cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SynchronousMachineConversion.java

```diff
@@ -70,10 +70,14 @@ public void convert() {
         }
         double normalPF = p.asDouble("normalPF");
         if (!Double.isNaN(normalPF)) {
-            g.newExtension(ActivePowerControlAdder.class)
-                    .withParticipate(normalPF != 0.0)
-                    .withParticipationFactor(normalPF)
-                    .add();
+            if (context.config().createActivePowerControlExtension()) {
+                g.newExtension(ActivePowerControlAdder.class)
+                        .withParticipate(normalPF != 0.0)
```

Comment:
we should probably change this to .withParticipate(true) always

CGMES does not have an equivalent for this participate attribute

then load flow implementations can use distribution proportional to Pmax, to targetP, to remaining margin, independently of this participate flag for iIDMs created from CGMES

Reply:
Yes, I was thinking about that too... One of our test is failing because only one generator has a `participating` to true.

Reply:
Formal documentation about CGMES participation factor (`normalPF`) only says it is used for distributed slack: _Generating unit economic participation factor.  The sum of the participation factors across generating units does not have to sum to one.  It is used for representing distributed slack participation factor. The attribute shall be a positive value or zero._

It seems correct to set the flag `participating` to false when `normalPF` is zero, but if that introduces noise, I agree on setting it always to `true`, and let the Load Flow engines decide later if the unit is effectively used for distributed slack looking the `participationFactor` value.

Reply:
Thanks @zamarrenolm for these clarifications and your approval :-)